Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -196,6 +198,37 @@ public void ConfigureServices(IServiceCollection services)
             }
 
             services.AddHealthCheck(Configuration.GetConnectionString("EdFi_Admin"), IsSqlServer(databaseEngine));
+
+            ConfigureScheduledJobs(services);
+        }
+
+        // The purpose of the ConfigureScheduleJobs method is to wire-up background tasks to be handled by quartz-scheduler.net
+        private void ConfigureScheduledJobs(IServiceCollection services)
+        {
+            var enabledDistinctScheduledJobs =
+                ApiSettings.ScheduledJobs
+                    .Where(a => a.IsEnabled)
+                    .GroupBy(a => a.Name)
+                    .Select(a => a.First()).ToList();
+
+            if (!enabledDistinctScheduledJobs.Any())
+            {
+                return;
+            }
+
+            IScheduledJobsConfiguratorFactory scheduledJobsConfiguratorFactory = new ScheduledJobsConfiguratorFactory();
+
+            services.AddQuartz(
+                q =>
+                {
+                    q.UseMicrosoftDependencyInjectionJobFactory();
+                    enabledDistinctScheduledJobs.ForEach(scheduledJobSetting => scheduledJobsConfiguratorFactory.GetScheduledJobsConfigurator(scheduledJobSetting.Name).AddScheduledJob(q, scheduledJobSetting));
```

Comment:
The `GetScheduledJobsConfigurator` can return a null value. Is this never going to happen because `enabledDistinctScheduledJobs` is guaranteed to contain a subset of available jobs, or could there be a mismatch from configuration and a null reference exception here?